guide/software.rst

```diff
@@ -5,21 +5,22 @@ The software you'll need to download and install on your computer in order to
 contribute varies between projects; please refer to the documentation for the
 project you want to contribute to for details.
 
-The following information is a generic description of software or tools that
-you'll most likely need regardless of the project you work on.
+Below is a list of some of the software you'll most likely need regardless of
+the project you work on.
 
-Operating Systems: Windows, Linux, or OS X?
--------------------------------------------
+Operating Systems: Windows, GNU/Linux, or macOS?
```

Comment:
You're right, these changes are definitely opinionated. I won't change it for now, but in the future one mention of the GNU Project would be nice since most people (myself included) would not have known the details behind Linux-based operating systems without someone else using the term.